test/helpers/spectron/advancedAudioSettings.js

```diff
@@ -0,0 +1,20 @@
+
+export async function getInputValueId(t, element, index) {
+    return (await t.context.app.client.$$(element))[index].ELEMENT;
+}
+
+export async function getInputValue(t, element, index = 0) {
+    const id = await getInputValueId(t, element, index);
+
+    return t.context.app.client
+    .elementIdElement(id, '.')
+    .getValue('input');
+}
+
+export async function getInputCheckboxValue(t, element, index = 0) {
+    const id = await getInputValueId(t, element, index);
+
+    return t.context.app.client
+    .elementIdElement(id, '.')
+    .isSelected('input');
+}
```

Comment:
This file's formatting needs cleaning up.  Please download the "Prettier" extension for your editor and run it on this file.

Reply:
Done.